[npm start] - Property 'media' is missing in type '{ matches: true; }'

# Error while building the site using `npm start`

> There is no error in my `npm install`

> When I run `npm start`, I get this error:

`ERROR in src/app/pages/component-sidenav/component-sidenav.spec.ts(24,49): error TS2345: Argument of type '{ matches: true; }' is not assignable to parameter of type 'MediaQueryList'. Property 'media' is missing in type '{ matches: true; }'.`

--- a/src/app/pages/component-sidenav/component-sidenav.spec.ts
+++ b/src/app/pages/component-sidenav/component-sidenav.spec.ts
@@ -23,7 +23,7 @@ describe('ComponentSidenav', () => {
     // Spy on window.mediaMatch and return stub
     spyOn(window, 'matchMedia').and.returnValue({
       matches: true
-    });
+    } as any);
 
     fixture.detectChanges();
     async(() => {

